packages/core/lib/debug/interpreter.js

```diff
@@ -761,6 +778,21 @@ class DebugInterpreter {
     }
     return tempPrintouts;
   }
+
+  // update the numbers of lines to be printed from options -<num>|+<num> from user args
+  updatePrintoutLines(userArgs, currentLines) {
```

Comment:
I think this function could use a better name... after all, it doesn't actually *update* anything, does it?

Reply:
how about parsePrintoutLines()?

Reply:
Makes sense!

Reply:
Hm, this seems to have been renamed in an inconsistent fashion.  Well -- I have a longer comment about that once the rest of the review is done, but noting it here also for now.